samples/openthread/ncp/src/user_vendor_hook.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+/**
+ * @file
+ *   This file shows how to implement the NCP vendor hook.
+ */
+#if OPENTHREAD_ENABLE_NCP_VENDOR_HOOK
+#include "nrf_802154_radio_wrapper.h"
+#include <logging/log.h>
+#include "ncp_base.hpp"
```

Comment:
These inclusions to OT headers should use angle brackets.

Reply:
Done